doc: clarify instruction for handling gcc-multilib on AArch64 systems

sheebzzz opened the pull request:
Fixed wording in Getting Started Guide: changed "remove" to "omit" for gcc-multilib on AArch64 systems to make instructions clearer. No functional changes.
Signed-off-by: Sheeba Babu <sheebakarickom@gmail.com>

henrikbrixandersen reviewed (dismissed):
Thanks, much clearer.

henrikbrixandersen commented:
@sheebzzz You need to fix the commit log and force push:

> 3: UC4 Commit message body line exceeds max length (79>75): "Changed "remove" to "omit" to clarify package instructions for AArch64 systems."

sheebzzz committed:
doc: clarify gcc-multilib instructions for AArch64


Changed "remove" to "omit" to clarify package instructions
for AArch64 systems. This improves clarity for users
following the Zephyr SDK setup guide.

Signed-off-by: Sheeba Babu <sheebakarickom@gmail.com>
